### ISSUE TITLE
Fix: a11y-test failure

### ISSUE THREAD

**Issue by maureenlholland** [mozilla/bedrock#16787]
## Description

 Axe-core® Accessibility Results for /en-US/advertising/ (desktop) project
Page URL: http://localhost:8000/en-US/advertising/?automation=true

Aside should not be contained in another landmark

From
```
<aside class="mozads-stat"> <span class="mozads-stat-text"> <span class="mozads-stat-number">200M+</span> <span class="mozads-stat-label">monthly active Firefox users</span> </span> </aside>

<aside class="mozads-stat"> <span class="mozads-stat-text"> <span class="mozads-stat-number">16.2M</span> <span class="mozads-stat-label">monthly active MDN users</span> </span> </aside>
```
---

#### Success Criteria
- [x] no aside contained in another landmark
- [x] a11y-tests passing

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/WT-307)
┆Priority: P2
┆Sprint: Backlog

**Comment by janbrasna** [comment]
https://github.com/mozilla/bedrock/actions/runs/18596655606 ✅
